docs/source/examples/mg-mode-part-i.ipynb

```diff
@@ -0,0 +1,926 @@
+    "mox7_data = openmc.Macroscopic('mox7')\n",
+    "mox87_data = openmc.Macroscopic('mox87')\n",
+    "fiss_chamber_data = openmc.Macroscopic('fiss_chamber')\n",
+    "guide_tube_data = openmc.Macroscopic('guide_tube')\n",
+    "water_data = openmc.Macroscopic('water')\n",
+    "\n",
+    "# Instantiate Materials dictionary\n",
+    "materials = {}\n",
+    "\n",
+    "# Instantiate some Materials and register the appropriate Macroscopic objects\n",
+    "materials['UO2'] = openmc.Material(name='UO2')\n",
+    "materials['UO2'].set_density('macro', 1.0)\n",
+    "materials['UO2'].add_macroscopic(uo2_data)\n",
+    "\n",
+    "materials['MOX 4.3%'] = openmc.Material(name='MOX 4.3%')\n",
+    "materials['MOX 4.3%'].set_density('macro', 1.0)\n",
+    "materials['MOX 4.3%'].add_macroscopic(mox43_data)\n",
+    "\n",
+    "materials['MOX 7.0%'] = openmc.Material(name='MOX 7.0%')\n",
+    "materials['MOX 7.0%'].set_density('macro', 1.0)\n",
+    "materials['MOX 7.0%'].add_macroscopic(mox7_data)\n",
+    "\n",
+    "materials['MOX 8.7%'] = openmc.Material(name='MOX 8.7%')\n",
+    "materials['MOX 8.7%'].set_density('macro', 1.0)\n",
+    "materials['MOX 8.7%'].add_macroscopic(mox87_data)\n",
+    "\n",
+    "materials['Fission Chamber'] = openmc.Material(name='Fission Chamber')\n",
+    "materials['Fission Chamber'].add_macroscopic(fiss_chamber_data)\n",
+    "\n",
+    "materials['Guide Tube'] = openmc.Material(name='Guide Tube')\n",
+    "materials['Guide Tube'].add_macroscopic(guide_tube_data)\n",
+    "\n",
+    "materials['Water'] = openmc.Material(name='Water')\n",
+    "materials['Water'].add_macroscopic(water_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can go ahead and produce a `materials.xml` file for use by OpenMC"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Instantiate a Materials collection, register all Materials, and export to XML\n",
+    "materials_file = openmc.Materials(materials.values())\n",
+    "\n",
+    "# Set the location of the cross sections file\n",
+    "materials_file.cross_sections = './mgxs.h5'\n",
+    "\n",
+    "materials_file.export_to_xml()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Our next step will be to create the geometry information needed for our assembly and to write that to the `geometry.xml` file.\n",
+    "\n",
+    "We will begin by defining the surfaces, cells, and universes needed for each of the individual fuel pins, guide tubes, and fission chambers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Create the surface used for each pin\n",
+    "pin_surf = openmc.ZCylinder(surface_id=1, x0=0, y0=0, R=0.54, name='pin_surf')\n",
```

Comment:
I did remove the unnecessary superfluous `surface_id` parameter, the users likely dont need to use that.